Makefile

```diff
@@ -0,0 +1,10 @@
+#/bin/bash
+
```

Comment:
I will add the `.travis.yml` file but will enhance it a little bit to also work with forks. See http://www.ruflin.com/2015/08/13/fix-for-travis-ci-failure-in-forked-golang-repositories/